widget/embedded/src/pages/WalletsPage.tsx

```diff
@@ -90,14 +90,17 @@ export function WalletsPage() {
   const countActiveCategories = getCountCategories(filteredBlockchains);
   const showCategory = countActiveCategories !== 1;
 
-  const filteredWallets = list.filter((wallet) => {
-    for (const type of wallet.blockchainTypes) {
-      if (filterByType(type, blockchainCategory)) {
-        return true;
-      }
-    }
-    return false;
-  });
+  const filteredWallets =
+    fetchMetaStatus === 'loading'
+      ? list
+      : list.filter((wallet) => {
+          for (const type of wallet.blockchainTypes) {
+            if (filterByType(type, blockchainCategory)) {
+              return true;
+            }
+          }
+          return false;
+        });
```

Comment:
I believe this condition alters the default behavior. If the meta request fails, the skeleton list will be empty.